src/main/core/Resources/modules/account/security/components/list.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import {PropTypes as T} from 'prop-types'
+
+import {trans} from '#/main/app/intl/translation'
+import {ListData} from '#/main/app/content/list/containers/data'
+
+const SecurityLogList = (props) =>
+  <ListData
+    name={props.name}
+    fetch={{
+      url: props.url,
+      autoload: true
+    }}
+    definition={[
+      {
+        name: 'doer',
+        type: 'user',
+        label: trans('user'),
+        displayed: true
+      }, {
+        name: 'date',
+        label: trans('date'),
+        type: 'date',
+        options: {time: true},
+        displayed: true
+      }, {
+        name: 'details',
+        type: 'string',
+        label: trans('details'),
+        displayed: true
+      }, {
+        name: 'target',
+        type: 'user',
+        label: trans('target'),
+        displayed: false
+      }, {
+        name: 'event',
+        type: 'translation',
+        label: trans('event'),
+        displayed: false,
+        options: {
+          domain: 'security'
+        }
+      }
+    ].concat(props.definition)}
```

Comment:
Not needed.
This is only used for reusable list components.

Reply:
removed, thx :)